updateUserFeatureFlag/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/models"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/go-playground/validator/v10"
+)
+
+var validate *validator.Validate
+
+func init() {
+	validate = validator.New()
+}
+
+func processUpdateById(userId string, flagId string, requestBody models.UpdateUserMapping) (*models.FeatureFlagUserMapping, error) {
+
+	db := database.CreateDynamoDB()
+
+	input := &dynamodb.UpdateItemInput{
+		TableName: aws.String(database.GetTableName(utils.FFUM_TABLE_NAME)),
+		Key: map[string]*dynamodb.AttributeValue{
+			"userId": {
+				S: aws.String(userId),
+			},
+			"flagId": {
+				S: aws.String(flagId),
+			},
+		},
+		UpdateExpression: aws.String("set #status = :status, #updatedBy = :updatedBy, #updatedAt = :updatedAt"),
```

Comment:
In this one thing, which I wanted to understand is, can we have few values those are optional and should not be updated when not passed? @shubham-y

Reply:
https://stackoverflow.com/questions/45842363/dynamodb-updateitem-ignore-null-values-in-expressionattributevalues this link can help

Reply:
> In this one thing, which I wanted to understand is, can we have few values those are optional and should not be updated when not passed? @shubham-y 

In this current scenario we do not need that because status and updatedBy will be required values to be passed in the update API